assignments/assignment-8/assignment-8.7.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Assignment 8.7</title>
+    </head>
+    <body>
+        <form>
+            <h2>Photo contect application form</h2>
+            <div>
+                <label for="fullNameInput">Full Name</label>
+                <input
+                    type="text"
+                    placeholder="Full Name"
+                    id="fullNameInput"
+                    name="fullName"
+                    required
+                />
+                <label for="emailInput">Email</label>
+                <input
+                    type="email"
+                    placeholder="Email"
+                    id="emailInput"
+                    name="email"
+                />
+            </div>
+
+            <label for="imageInput">Upload your photo</label>
+            <input
+                type="file"
+                required
+                id="imageInput"
+                name="personalPhoto"
+                accept="image/jpeg"
+            />
+            <label for="agreementCheckboxInput"
+                >I agree the terms and conditions</label
+            >
+            <input
+                type="checkbox"
+                id="agreementCheckboxInput"
```

Comment:
you can drop "Input" in checkbox and radio input types for id,for attr. Like: id="agreementCheckbox"

Reply:
yeah good point